api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -393,15 +397,16 @@ private boolean fieldChanged(Diff diff, String field) {
    * @return true if the Profile is new
    */
   private boolean isNewProfile(Diff diff) {
-    final Predicate<Change> newProfilePred =
-        change ->
-            change instanceof NewObject
-                && change
-                    .getAffectedGlobalId()
-                    .getTypeName()
-                    .equals(Profile.class.getCanonicalName());
 
-    return diff.getChanges(newProfilePred).size() > 0;
+    return diff.getChanges(
```

Comment:
Why change this?  The linter-required formatting makes this version harder to follow

Reply:
I thought lambdas were preferred in this kind of usage and thought you might not have realized you can use a lambda instead of declaring a predicate. Arguably we could improve it by  declaring the list of changes and then taking its size. I believe  IntelliJ suggested this change as well.

Reply:
Makes sense.  Lambda + separated size check sounds like the best option here.  Thanks.

It's certainly not a priority to make that change.